.clang-format

```diff
@@ -1,6 +1,6 @@
 Language: Cpp
 # BasedOnStyle:  LLVM
-AccessModifierOffset: 0
+AccessModifierOffset: -4
```

Comment:
This one we can rollback if you dont agree. I prefer acces modifiers to be aligned on left like this:

(After this change):
```cpp
class Test {
private:
    int var = 0;
};
```

(Previously):
```cpp
class Test {
    private:
    int var = 0;
};
```